package.json

```diff
@@ -31,6 +31,7 @@
     "clean": "hardhat clean && hardhat clean --config hardhat.config.zksync.ts"
   },
   "dependencies": {
+    "@defi-wonderland/smock": "2.3.5",
```

Comment:
AFAIK, by including this in the `dependencies` section will increase the size of the public npm package. Do we need these dependencies apart from the development phases?

Reply:
[51c0702](https://github.com/VenusProtocol/venus-protocol/pull/506/commits/51c070278168a0a65789c3b9e6320a1f57d5876c)

Reply:
What about the dependency with `"hardhat": "2.19.5"`?

Reply:
We need this to pass CI checks

Reply:
https://github.com/VenusProtocol/venus-protocol/pull/506/commits/4bd4d30182e490554c7d132bda0853bc68ce8d89